package.json

```diff
@@ -78,4 +79,4 @@
     "async": "^1.5.0",
     "npmi": "^1.0.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
I agree that it feels a little weird. The goal was to codify the [semver conventions](https://github.com/cfpb/capital-framework/issues/179). Those conventions say "when X happens to a component's version, do Y to CF's version". My thinking was that we'll *never* be manually updating CF's version, it'll only change when a component changes. So I automated the CF manifest logic and left the component manifests to be update by humans.